crates/client/settlement/src/errors.rs

```diff
@@ -0,0 +1,30 @@
+use std::time::Duration;
+
+use async_trait::async_trait;
+use futures_timer::Delay;
+use thiserror::Error;
+
+use crate::RetryStrategy;
+
+#[derive(Debug, Error)]
+pub enum UnrecoverableError {
+    #[error("State root mismatch for level {0}")]
```

Comment:
what does `level` means in this context?

Reply:
The Starkness level where state root differs from what we got from the settlement contract

Reply:
Yes, I'll rename